src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserHelperClasses.cs

```diff
@@ -318,21 +326,28 @@ internal static SqlEnvChange Allocate()
                     return item;
                 }
             }
+            // if we didn't find an item in the pool we've either never craete any or we're under pressure 
```

Comment:
`craete`

Reply:
You wouldn't believe I'm an English monoglot based on my comments would you 😁

Reply:
Oh yes, I would, believe me :D